base/tomcat-9.0/conf/server.xml

```diff
@@ -126,7 +126,6 @@
                ocspMinCacheEntryDuration="7200"
                ocspMaxCacheEntryDuration="14400"
                ocspTimeout="10"
-               serverCertNickFile="[PKI_INSTANCE_PATH]/conf/serverCertNick.conf"
```

Comment:
It is OK to remove the default value but I am not sure about remove this configuration for the moment.

Reply:
The attribute will be added when needed, so it should be fine.